docs/APIRef.ActionsOnElement.md

```diff
@@ -107,3 +108,10 @@ await element(by.id('scrollView')).swipe('down');
 await element(by.id('scrollView')).swipe('down', 'fast');
 await element(by.id('scrollView')).swipe('down', 'fast', 0.5);
 ```
+
+### `scrollDatePickerAtColumnToValue(column,value)`
+Scroll Date Picker at defined column to required value
+
+```js
+await element(by.type('UIPickerView')).scrollDatePickerAtColumnToValue(0,'22');
```

Comment:
1. Please add a valid test screen in the test project, add both date and time pickers, and add a test to verify the API works.
2. Android implementation needs to have a different API. We can not use same API for both, Android has only `date` or `time` pickers, iOS has `date`, `time` and `datetime` pickers, therefore:
iOS API should use `setValueAtColumn`
Android API should use `setDate(y, m, d)` and `setTime(h, m)`

Android resources:
  [DatePicker](https://developer.android.com/reference/android/widget/TimePicker.html)
  [TimePicker](https://developer.android.com/reference/android/widget/DatePicker.html)
Espresso API: 
  [PickerActions](https://developer.android.com/reference/android/support/test/espresso/contrib/PickerActions.html)
  [Example](https://stackoverflow.com/questions/44835094/check-datepicker-calendar-value-in-android-espresso-framework)

Reply:
Let's make the Android API more consistent with iOS:

`setToDate(y, m, d)` and `setToTime(h, m)`